Automattic/vip-design-system
# LinkExternal: migrate from dashboard

## Summary

We're planning to migrate the footer from the dashboard (#373), which contains the  `LinkExternal` component. To make that migration easier, I figured I'd start here. 

## Details

In the first commit https://github.com/Automattic/vip-design-system/commit/0ecdce962fda7504bb8ed9e6c97394a48b26cd0a, it's 1:1 with the dashboard, just for ease of comparing the following changes in subsequent commits. But mostly, I've kept the component the same. I checked in with @djalmaaraujo to see if we wanted to use this as an opportunity to change anything about the component or if we want to keep it the same for a smoother migration. I initially thought we might want an `isExternal` prop on the existing `<Link/>` component, but I want to ensure I follow the standards set here. 🙂 

## Checklist


## Steps to Test

1. Pull down PR.
2. `npm run dev`.
3. Open Storybook.
4. Navigate to `LinkExternal` component
5. Verify component works the same as the dashboard component
